### PR TITLE
fix asahi.com placeholder

### PR DESCRIPTION
**Pull request by Yuki2718**
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: 

URL: `https://www.asahi.com/sp/articles/ASP1J6SM2P1JUSPT007.html?iref=sptop_7_01`
Placeholder

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)
<details><summary>Screenshots:</summary>

asahi1

asahi2

asahi_pc1

asahi_pc2

</details><br/>

***Steps to reproduce the problem***:

Visit the site each on mobile and on PC.

***System configuration***

**Filters:**

Base, Mobile, TPL, and Japanese

[//]: # (Please enter the correct values for your case to the table below)

Information                            | Value
---                                    | ---
Operating system:                      | Win 10
Browser:                               | Firefox 84.0.2 (mobile UA) / Brave 1.18.78
AdGuard version:                       | 3.5.31
AdGuard DNS:                           | None
Stealth mode options (Windows only)    | Disabled
Helpdesk ID (if exists):               | ?

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

### CODE REVIEW

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -82,6 +82,7 @@ mindhack2ch.com#?#.article-body > .ad:has(> .adsbygoogle)
 mindhack2ch.com#?#.article-body > .rss:has(> script)
 mindhack2ch.com#?#.article-post-bottom > .rss-article:has(> .rss-article-item)
 !
+asahi.com#?##RelatedNews > .InnerArticleList > li:has(> div[id^="div-gpt-ad"])
```
Comment (zloyden) [inline JapaneseFilter/sections/css_extended.txt:85]: There is missed tag `ul` in `.InnerArticleList`. We avoid tags if it's necessary.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/css_extended.txt (hunk 1 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/css_extended.txt:85]: You mean I should add `ul`?

Comment (zloyden) [reply JapaneseFilter/sections/css_extended.txt:85]: Yes.

Comment (Yuki2718) [reply JapaneseFilter/sections/css_extended.txt]: Added.

**Review by zloyden — APPROVED** [review]

**Pull request merged**